websites/O/ogladajanime/presence.ts

```diff
@@ -0,0 +1,43 @@
+const presence = new Presence({ clientId: "1137362720254074972" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	anime: HTMLElement = document.querySelector("#anime_name_id"),
+	animeicon = document.querySelector(".img-fluid.lozad");
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		startTimestamp: browsingTimestamp,
+		largeImageKey: "https://i.imgur.com/xHce23t.png",
+	};
+
+	if (document.location.pathname === "/main2")
+		presenceData.details = "Viewing the home page";
+	else if (document.location.pathname.includes("/search/name/"))
```

Comment:
To make the code more readable/concise, destructure properties of `document.location`
```ts
const { pathname, href } = document.location;
```

Reply:
This one is not resolved yet.